admin-dev/themes/default/js/bundle/product/form.js

```diff
@@ -874,7 +873,7 @@ var specificPrices = (function() {
 						<td>'+ specific_price.impact +'</td>\
 						<td>'+ specific_price.period +'</td>\
 						<td>'+ specific_price.from_quantity +'</td>\
```

Comment:
Multiline strings in JS are [not safe](https://google.github.io/styleguide/javascriptguide.xml#Multiline_string_literals), you could use transpiled ES6 and [template strings](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/template_strings) instead.